ui/imports/shared/views/chat/MessageView.qml

```diff
@@ -71,7 +71,12 @@ Loader {
             return []
         const separator = " "
         const arr = links.split(separator)
-        const filtered = arr.filter(v => v.toLowerCase().endsWith('.gif'))
+        const filtered = arr.filter(value => {
```

Comment:
I'm leaving this here for the record. I think it can also be fixed in another PR where we clean old settings. There's a conflict between the new/old settings.
E.g. GIF enabled in the new settings and disabled in the old settings:

<img width="665" alt="Screenshot 2023-10-16 at 15 53 51" src="https://github.com/status-im/status-desktop/assets/47811206/cf3372ce-c2ec-456e-b969-ca17677f723c">